Build with cpp bindings fails: `configure: error: cannot find required auxiliary files: ltmain.sh compile missing`

**Describe the bug**
Build fails due to missing scripts in bindings/cpp. Apparently broken by https://github.com/PortAudio/portaudio/commit/4d1eb3c6bb87f328a07cb0dd0ea38973cb943fc6

**To Reproduce**
Run the build with cpp enabled.

**Expected behavior**
It should succeed, as it was earier.

**Actual behavior**
```
--->  Configuring portaudio

Executing:  cd "/opt/local/var/macports/build/portaudio-f8953ce0/work/portaudio-7786be7f5f2f8f908ec66d1dc3ab8f3e61562f9f" && autoreconf -fiv

autoreconf: export WARNINGS=

autoreconf: Entering directory '.'

autoreconf: configure.ac: not using Gettext

autoreconf: running: /opt/local/bin/aclocal --force

autoreconf: configure.ac: tracing

autoreconf: configure.ac: adding subdirectory bindings/cpp to autoreconf

autoreconf: Entering directory 'bindings/cpp'

autoreconf: configure.ac: not using Gettext

autoreconf: running: /opt/local/bin/aclocal --force

autoreconf: configure.ac: tracing

autoreconf: running: /opt/local/bin/glibtoolize --copy --force

glibtoolize: putting auxiliary files in '../..'.

glibtoolize: copying file '../../ltmain.sh'

glibtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,

glibtoolize: and rerunning glibtoolize and aclocal.

glibtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

autoreconf: configure.ac: not using Intltool

autoreconf: configure.ac: not using Gtkdoc

autoreconf: running: /opt/local/bin/aclocal --force

autoreconf: running: /opt/local/bin/autoconf --force

autoreconf: configure.ac: not using Autoheader

autoreconf: running: /opt/local/bin/automake --add-missing --copy --force-missing

configure.ac:35: installing '../../compile'

configure.ac:17: installing '../../missing'

bin/Makefile.am: installing '../../depcomp'

autoreconf: './install-sh' is created

autoreconf: './config.sub' is created

autoreconf: './config.guess' is created

autoreconf: Leaving directory 'bindings/cpp'

autoreconf: running: /opt/local/bin/glibtoolize --copy --force

glibtoolize: putting auxiliary files in '.'.

glibtoolize: copying file './ltmain.sh'

glibtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,

glibtoolize: and rerunning glibtoolize and aclocal.

glibtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

autoreconf: configure.ac: not using Intltool

autoreconf: configure.ac: not using Gtkdoc

autoreconf: running: /opt/local/bin/aclocal --force

autoreconf: running: /opt/local/bin/autoconf --force

configure.ac:519: warning: AC_C_BIGENDIAN should be used with AC_CONFIG_HEADERS

autoreconf: configure.ac: not using Autoheader

autoreconf: configure.ac: not using Automake

autoreconf: Leaving directory '.'

Executing:  cd "/opt/local/var/macports/build/portaudio-f8953ce0/work/portaudio-7786be7f5f2f8f908ec66d1dc3ab8f3e61562f9f" && ./configure --prefix=/opt/local --disable-mac-universal --enable-cxx --enable-option-checking --without-asihpi --without-winapi --without-alsa --without-jack --without-oss --without-pulseaudio

checking build system type... powerpc-apple-darwin10.8.0

checking host system type... powerpc-apple-darwin10.8.0

checking target system type... powerpc-apple-darwin10.8.0

checking for gcc... /opt/local/bin/gcc-mp-14

checking whether the C compiler works... yes

checking for C compiler default output file name... a.out

checking for suffix of executables...

checking whether we are cross compiling... no

checking for suffix of object files... o

checking whether the compiler supports GNU C... yes

checking whether /opt/local/bin/gcc-mp-14 accepts -g... yes

checking for /opt/local/bin/gcc-mp-14 option to enable C11 features... none needed

checking whether the compiler supports GNU C++... yes

checking whether /opt/local/bin/g++-mp-14 accepts -g... yes

checking for /opt/local/bin/g++-mp-14 option to enable C++11 features... none needed

checking how to print strings... printf

checking for a sed that does not truncate output... /usr/bin/sed

checking for grep that handles long lines and -e... /usr/bin/grep

checking for egrep... /usr/bin/grep -E

checking for fgrep... /usr/bin/grep -F

checking for ld used by /opt/local/bin/gcc-mp-14... /opt/local/bin/ld

checking if the linker (/opt/local/bin/ld) is GNU ld... no

checking for BSD- or MS-compatible name lister (nm)... /opt/local/bin/nm

checking the name lister (/opt/local/bin/nm) interface... BSD nm

checking whether ln -s works... yes

checking the maximum length of command line arguments... 196608

checking how to convert powerpc-apple-darwin10.8.0 file names to powerpc-apple-darwin10.8.0 format... func_convert_file_noop

checking how to convert powerpc-apple-darwin10.8.0 file names to toolchain format... func_convert_file_noop

checking for /opt/local/bin/ld option to reload object files... -r

checking for file... file

checking for objdump... no

checking how to recognize dependent libraries... pass_all

checking for dlltool... no

checking how to associate runtime and link libraries... printf %s\n

checking for ranlib... ranlib

checking for ar... ar

checking for archiver @FILE support... no

checking for strip... strip

checking for gawk... no

checking for mawk... no

checking for nawk... no

checking for awk... awk

checking command to parse /opt/local/bin/nm output from /opt/local/bin/gcc-mp-14 object... ok

checking for sysroot... no

checking for a working dd... /bin/dd

checking how to truncate binary pipes... /bin/dd bs=4096 count=1

checking for mt... no

checking if : is a manifest tool... no

checking for dsymutil... dsymutil

checking for nmedit... nmedit

checking for lipo... lipo

checking for otool... otool

checking for otool64... no

checking for -single_module linker flag... yes

checking for -no_fixup_chains linker flag... no

checking for -exported_symbols_list linker flag... yes

checking for -force_load linker flag... yes

checking for stdio.h... yes

checking for stdlib.h... yes

checking for string.h... yes

checking for inttypes.h... yes

checking for stdint.h... yes

checking for strings.h... yes

checking for sys/stat.h... yes

checking for sys/types.h... yes

checking for unistd.h... yes

checking for dlfcn.h... yes

checking for objdir... .libs

checking if /opt/local/bin/gcc-mp-14 supports -fno-rtti -fno-exceptions... no

checking for /opt/local/bin/gcc-mp-14 option to produce PIC... -fno-common -DPIC

checking if /opt/local/bin/gcc-mp-14 PIC flag -fno-common -DPIC works... yes

checking if /opt/local/bin/gcc-mp-14 static flag -static works... no

checking if /opt/local/bin/gcc-mp-14 supports -c -o file.o... yes

checking if /opt/local/bin/gcc-mp-14 supports -c -o file.o... (cached) yes

checking whether the /opt/local/bin/gcc-mp-14 linker (/opt/local/bin/ld) supports shared libraries... yes

checking dynamic linker characteristics... darwin10.8.0 dyld

checking how to hardcode library paths into programs... immediate

checking whether stripping libraries is possible... yes

checking if libtool supports shared libraries... yes

checking whether to build shared libraries... yes

checking whether to build static libraries... yes

checking how to run the C++ preprocessor... /opt/local/bin/g++-mp-14 -E

checking for ld used by /opt/local/bin/g++-mp-14... /opt/local/bin/ld

checking if the linker (/opt/local/bin/ld) is GNU ld... no

checking whether the /opt/local/bin/g++-mp-14 linker (/opt/local/bin/ld) supports shared libraries... yes

checking for /opt/local/bin/g++-mp-14 option to produce PIC... -fno-common -DPIC

checking if /opt/local/bin/g++-mp-14 PIC flag -fno-common -DPIC works... yes

checking if /opt/local/bin/g++-mp-14 static flag -static works... no

checking if /opt/local/bin/g++-mp-14 supports -c -o file.o... yes

checking if /opt/local/bin/g++-mp-14 supports -c -o file.o... (cached) yes

checking whether the /opt/local/bin/g++-mp-14 linker (/opt/local/bin/ld) supports shared libraries... yes

checking dynamic linker characteristics... darwin10.8.0 dyld

checking how to hardcode library paths into programs... immediate

checking for a BSD-compatible install... /usr/bin/install -c

checking whether ln -s works... yes

checking for ar... /opt/local/bin/ar

checking whether byte ordering is bigendian... yes

checking for pkg-config... /opt/local/bin/pkg-config

checking pkg-config is at least version 0.9.0... yes

checking for sndio... no

checking for sys/audioio.h... no

checking size of short... 2

checking size of int... 4

checking size of long... 4

checking for clock_gettime in -lrt... no

checking for clock_gettime... no

checking for nanosleep... yes

checking whether the C compiler accepts -std=c11... yes

configure: creating ./config.status

config.status: creating Makefile

config.status: creating portaudio-2.0.pc

config.status: executing libtool commands

=== configuring in bindings/cpp (/opt/local/var/macports/build/portaudio-f8953ce0/work/portaudio-7786be7f5f2f8f908ec66d1dc3ab8f3e61562f9f/bindings/cpp)

configure: running /bin/sh ./configure --disable-option-checking '--prefix=/opt/local'  '--disable-mac-universal' '--enable-cxx' '--enable-option-checking' '--without-asihpi' '--without-winapi' '--without-alsa' '--without-jack' '--without-oss' '--without-pulseaudio' 'CC=/opt/local/bin/gcc-mp-14' 'CFLAGS=-pipe -Os -arch ppc' 'LDFLAGS=-L/opt/local/lib -Wl,-headerpad_max_install_names -arch ppc' 'CPPFLAGS=-I/opt/local/include' 'CXX=/opt/local/bin/g++-mp-14' 'CXXFLAGS=-pipe -Os -arch ppc' --cache-file=/dev/null --srcdir=.

configure: WARNING: unrecognized options: --disable-mac-universal, --enable-cxx, --without-asihpi, --without-winapi, --without-alsa, --without-jack, --without-oss, --without-pulseaudio

configure: error: cannot find required auxiliary files: ltmain.sh compile missing

configure: error: ./configure failed for bindings/cpp

Command failed:  cd "/opt/local/var/macports/build/portaudio-f8953ce0/work/portaudio-7786be7f5f2f8f908ec66d1dc3ab8f3e61562f9f" && ./configure --prefix=/opt/local --disable-mac-universal --enable-cxx --enable-option-checking --without-asihpi --without-winapi --without-alsa --without-jack --without-oss --without-pulseaudio

Exit code: 1
```
It fails without autotools (just running configure) as well. `autoconf-archive` is installed.

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version: 10.6.8
 - PortAudio version: stable, nightly snapshot (which?), current (please give date and/or Git hash): 7786be7f5f2f8f908ec66d1dc3ab8f3e61562f9f

**Additional context**
MacPorts env

@gruenich Could you please take a look?

--- a/bindings/cpp/Makefile.in
+++ b/bindings/cpp/Makefile.in
@@ -184,8 +184,10 @@ am__define_uniq_tagged_files = \
   done | $(am__uniquify_input)`
 DIST_SUBDIRS = $(SUBDIRS)
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/portaudiocpp.pc.in \
-	AUTHORS COPYING ChangeLog INSTALL NEWS README compile \
-	config.guess config.sub install-sh ltmain.sh missing
+	$(top_srcdir)/../../compile $(top_srcdir)/../../config.guess \
+	$(top_srcdir)/../../config.sub $(top_srcdir)/../../install-sh \
+	$(top_srcdir)/../../ltmain.sh $(top_srcdir)/../../missing \
+	AUTHORS COPYING ChangeLog INSTALL NEWS README
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 distdir = $(PACKAGE)-$(VERSION)
 top_distdir = $(distdir)

--- a/bindings/cpp/bin/Makefile.in
+++ b/bindings/cpp/bin/Makefile.in
@@ -128,7 +128,7 @@ AM_V_at = $(am__v_at_@AM_V@)
 am__v_at_ = $(am__v_at_@AM_DEFAULT_V@)
 am__v_at_0 = @
 am__v_at_1 = 
-depcomp = $(SHELL) $(top_srcdir)/depcomp
+depcomp = $(SHELL) $(top_srcdir)/../../depcomp
 am__maybe_remake_depfiles = depfiles
 am__depfiles_remade = $(BINDIR)/$(DEPDIR)/devs.Po \
 	$(BINDIR)/$(DEPDIR)/sine.Po
@@ -175,7 +175,7 @@ am__define_uniq_tagged_files = \
   unique=`for i in $$list; do \
     if test -f "$$i"; then echo $$i; else echo $(srcdir)/$$i; fi; \
   done | $(am__uniquify_input)`
-am__DIST_COMMON = $(srcdir)/Makefile.in $(top_srcdir)/depcomp
+am__DIST_COMMON = $(srcdir)/Makefile.in $(top_srcdir)/../../depcomp
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 ACLOCAL = @ACLOCAL@
 AMTAR = @AMTAR@

--- a/bindings/cpp/lib/Makefile.in
+++ b/bindings/cpp/lib/Makefile.in
@@ -162,7 +162,7 @@ AM_V_at = $(am__v_at_@AM_V@)
 am__v_at_ = $(am__v_at_@AM_DEFAULT_V@)
 am__v_at_0 = @
 am__v_at_1 = 
-depcomp = $(SHELL) $(top_srcdir)/depcomp
+depcomp = $(SHELL) $(top_srcdir)/../../depcomp
 am__maybe_remake_depfiles = depfiles
 am__depfiles_remade = $(SRCDIR)/$(DEPDIR)/BlockingStream.Plo \
 	$(SRCDIR)/$(DEPDIR)/CFunCallbackStream.Plo \
@@ -223,7 +223,7 @@ am__define_uniq_tagged_files = \
   unique=`for i in $$list; do \
     if test -f "$$i"; then echo $$i; else echo $(srcdir)/$$i; fi; \
   done | $(am__uniquify_input)`
-am__DIST_COMMON = $(srcdir)/Makefile.in $(top_srcdir)/depcomp
+am__DIST_COMMON = $(srcdir)/Makefile.in $(top_srcdir)/../../depcomp
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 ACLOCAL = @ACLOCAL@
 AMTAR = @AMTAR@

